meeting_timesheet_analytic/README.rst

```diff
@@ -0,0 +1,29 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==========================
+Meeting timesheet analytic
+==========================
+
+* When the meeting is realized, inputs time sheet hours.
```

Comment:
Cuando la reunión se da cuenta, introduce partes de hora

When the meeting is set done, it creates timesheets.